e2e_test.py

```diff
@@ -0,0 +1,100 @@
+import os
+from collections import defaultdict
+import jsonlines
+import sys
+
+from colbert import Indexer, Searcher
+from colbert.infra import ColBERTConfig, RunConfig, Run
+from colbert.data import Queries
+
+CKPT = "/dfs/scratch0/okhattab/share/2021/checkpoints/msmarco.psg.kldR2.nway64.ib__colbert-400000"
+STACKEXCHANGE_DATA_PATH = "/future/u/xmgui/testdata"
+EXPERIMENT_DIR = "/future/u/xmgui/ColBERT/experiments"
+
+
+def build_index(dataset_path):
+    nbits = 2  # encode each dimension with 2 bits
```

Comment:
Maybe we can use 1 bit and max doclen of 180 to make the test faster - this might affect any accuracy checks we do though